### DISCUSSION TITLE
v0.27.2

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/UnblockNeteaseMusic#1135] [Announcements]
## TL; DR;
- Fixed compatibility with the latest NCM clients
- Other fixes and improvements

## Fixes
- [fix: local vip on newer versions](https://github.com/UnblockNeteaseMusic/server/commit/47dc62c24fc105e6cb46fd3099b94512b850e756) #1046
- [fix(hook): fix download error](https://github.com/UnblockNeteaseMusic/server/commit/91900ee756147d97f70fcd3cfbe2b753f4177cfa) #1065
- [fix(youtube.js): fix Cannot read properties of undefined (reading 'videoId')](https://github.com/UnblockNeteaseMusic/server/commit/73f1537acc7d048751f0442171402104a629f2b2) #1085

## Features
- [feat(app): enable endpoint by default](https://github.com/UnblockNeteaseMusic/server/commit/a6a62cfbe537c517f0db8b4dc87479c11db2a98d)

## Docs
- [docs: add execute method (RevivedUnblockInstaller)](https://github.com/UnblockNeteaseMusic/server/commit/5bff799d8e63e2554b979c63bd54827d171e1aef) #1049

## Chore
- Update dependencies to the latest

## New Contributors
* @ReviveUnblockNCMInstaller made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1049
* @rsp4jack made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1046
* @rankanin made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1085

**Full Changelog**: https://github.com/UnblockNeteaseMusic/server/compare/v0.27.1...v0.27.2

<hr /><em>This discussion was created from the release <a href='https://github.com/UnblockNeteaseMusic/server/releases/tag/v0.27.2'>v0.27.2</a>.</em>
